ts/features/itwallet/identification/screens/spid/ItwSpidIdpLoginScreen.tsx

```diff
@@ -41,6 +37,11 @@ const ItwSpidIdpLoginScreen = () => {
   const spidAuthUrl =
     ItwEidIssuanceMachineContext.useSelector(selectAuthUrlOption);
   const machineRef = ItwEidIssuanceMachineContext.useActorRef();
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
Adds more context. What do you think?
```suggestion
  const [isWebViewLoading, setWebViewLoading] = useState(true);
```

Reply:
Yes, it's okay.